src/coloris.js

```diff
@@ -149,7 +149,8 @@
     // Show the color picker on click on the input fields that match the selector
     addListener(document, 'click', selector, event => {
       const parent = settings.parent;
-      const coords = event.target.getBoundingClientRect();
+      // When input is of type hidden the bounding client rect is always 0
+      const coords = event.target.type === 'hidden' ? event.target.parent.getBoundingClientRect() : event.target.getBoundingClientRect();
```

Comment:
Binding the picker to a input of type `hidden` is not supported. Also there is no guarantee that the parent of the hidden input has the correct position, it may well be a `form` element that's much larger than a typical input field. This only works in your unique use case